### DISCUSSION TITLE
Error: SequelizeDatabaseError: Unknown column 'tbl_category.updatedAt`undefined' in 'order clause'

### DISCUSSION THREAD

**Discussion by shreyash0011** [sequelize/sequelize#15418] [Help & Questions]
basically I realize that this error came from updatedAt and createdAt not getting auto generated by default how to fix it.

**Comment by ViniciusMarchi** [comment]
The error is saying that you're trying to order query results by a column that do not exist in your `category` table. You are trying to order by the `updatedAt` column. To fix this issue you can explicitly specify that you want the `updatedAt` and `createdAt` columns to be generated by Sequelize. To do this set the `timestamps` option in your table model to `true`, like this example, modified from [Sequelize timestamps documentation](https://sequelize.org/docs/v6/core-concepts/model-basics/#timestamps)

```javascript
sequelize.define('User', {
  // ... (attributes)
}, {
  timestamps: true
});
```
However, this is unexpected behavior because, as the Sequelize timestamps documentation also show, these fields should be automatically generated if you don't change this behavior explicitly, as the quote below shows:

>By default, Sequelize automatically adds the fields createdAt and updatedAt to every model.

So, to get a better answer to your problem, it is very important to detail your question with code snippets, allowing the problem to be replicated, but i hope this anwser helps.
